components/builder-api/src/server/mod.rs

```diff
@@ -41,23 +41,24 @@ impl HttpGateway for ApiSrv {
     fn add_middleware(config: Arc<Self::Config>, chain: &mut iron::Chain) {
         chain.link(persistent::Read::<Self::Config>::both(config.clone()));
 
-        chain.link(persistent::Read::<OAuthCli>::both(OAuth2Client::new(
-            config.oauth.clone(),
-        )));
+        chain.link(persistent::Read::<OAuthCli>::both(
```

Comment:
Need to rebase w/ rustfmt 0.38 formatting?

Reply:
I think I'm on 0.9.0. I can pin back to 0.3.8 but is there a reason this part of the codebase is pinned back this far?

Reply:
0.3.8 is actually newer (go figure :)

Reply:
WHAT. Ugh, I'm all turned around. I'll get this updated then!

Reply:
This is corrected, I got my workstation all updated XD